src/main/resources/schema/input.graphqls

```diff
@@ -1,5 +1,8 @@
-input WishInput {
+
+input WishInput{
+  author: ID!
   title: String!
-  content: String!
+  description: String!
+  tags: String!
```

Comment:
Tags should be of array type

Reply:
Probably just an array of strings

Reply:
Sorry, I missed that.... Thanks for the correction